airqo-platform/AirQo-frontend
# Reimplement favorite Places using Api.

#### Summary of Changes (What does this PR do?)

- Utilizing the favorite Places Api in the mobile App.

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Checkout the branch
- Run the app
- Attempt to add or remove favorite places.
- Results should be consistent with results in the API endpoint for [getting Favorite places,](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-51eab2b9-5920-414e-9019-82781da58a62?ctx=documentation)

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
